build/package/Microsoft.DotNet.Cli.Installer.DEB.targets

```diff
@@ -15,7 +15,7 @@
       <DotnetDebToolProjectJsonLines Include="{" />
       <DotnetDebToolProjectJsonLines Include="%20%20%20%20&quot;version&quot;: &quot;1.0.0-*&quot;," />
       <DotnetDebToolProjectJsonLines Include="%20%20%20%20&quot;name&quot;: &quot;$(DotnetDebToolConsumerProjectName)&quot;," />
-      <DotnetDebToolProjectJsonLines Include="%20%20%20%20&quot;frameworks&quot;: { &quot;netcoreapp1.0&quot;: {} }," />
+      <DotnetDebToolProjectJsonLines Include="%20%20%20%20&quot;frameworks&quot;: { &quot;netcoreapp1.0&quot;: {}, &quot;netcoreapp1.1&quot;: {} }," />
```

Comment:
I assumed so. I tried this earlier but it didn't solve the Ubuntu installer issue.